packages/itwinui-css/backstop/tests/slider.html

```diff
@@ -142,17 +157,39 @@ <h2>Default</h2>
             </output>
             <div class="iui-slider-ticks">
               <!-- You could generate the ticks based on your min, max & step values. -->
-              <span class="iui-slider-tick">0</span>
-              <span class="iui-slider-tick">10</span>
-              <span class="iui-slider-tick">20</span>
-              <span class="iui-slider-tick">30</span>
-              <span class="iui-slider-tick">40</span>
-              <span class="iui-slider-tick">50</span>
-              <span class="iui-slider-tick">60</span>
-              <span class="iui-slider-tick">70</span>
-              <span class="iui-slider-tick">80</span>
-              <span class="iui-slider-tick">90</span>
-              <span class="iui-slider-tick">100</span>
+              <div class="iui-slider-tick">
+                <span class="iui-slider-tick-mark"></span><span class="iui-slider-tick-number">0</span>
```

Comment:
You might be able to remove `<span class="iui-slider-tick-mark"></span>` & use a pseudo element (`::before`) instead.

Reply:
Thanks for the idea, I removed the `iui-slider-tick-mark` span and reverted back to the old tick mark HTML structure. I replaced the `iui-slider-tick-mark` span with a `::before` on the `iui-slider-tick`.

#### Old (reverted back to this) `iui-slider-tick` structure
```html
<span class="iui-slider-tick">0</span>
```

#### New (deleted this and reverted back to the above version) `iui-slider-tick` structure
```html
<div class="iui-slider-tick">
  <span class="iui-slider-tick-mark"></span><span class="iui-slider-tick-number">0</span>
</div>
```

Reply:
Awesome, looks great & good use of a pseudo element!

Reply:
@FlyersPh9 Great, thanks! If there aren't any other changes that you recommend, can you please give your approval so that I can merge the branch? Thanks